infrastructure/ansible/roles/http_server/defaults/main.yml

```diff
@@ -14,7 +14,6 @@ http_server_db_port: "5432"
 
 http_server_db_name: "{{ lobby_db_name }}"
 http_server_db_user: "{{ lobby_db_user }}"
-http_server_db_password: "{{ lobby_db_password }}"
 
-## TODO: encrypt all of the below variables
-http_server_bcrypt_salt: $2a$10$IhIXWg4HkQRWrZqjj9kV0u
+create_issues_github_api_token: test
```

Comment:
Test?

Reply:
Note the value in the configuration.yml used for local development and automated tests:
- https://github.com/triplea-game/triplea/blob/master/http-server/configuration.yml#L6

Note that the value is specific, used as a magic flag to not send API requests if the value is equal to test:
- https://github.com/triplea-game/triplea/blob/master/http-clients/src/main/java/org/triplea/http/client/github/issues/GithubIssueClient.java
- https://github.com/triplea-game/triplea/blob/master/http-server/src/main/java/org/triplea/server/error/reporting/CreateIssueStrategy.java#L41

With this PR, note that the vagrant/prerelease/production configuration.yml is parameterized for this value:
- https://github.com/triplea-game/triplea/pull/5852/files#diff-448e030bac09cb5b6eaf84f89d4b5d54R2

The value we have here is the default, so we will use that unless there is an override present. That is intentional so that 'vagrant' will work without any overrides.

Then finally note the 'linode' group_var override that will override this for linode environments:
- https://github.com/triplea-game/triplea/pull/5852/files#diff-8a634d36d006538798cff653e9e28b53R18

Finally, we update inventory to define prerelease and production as children of 'linode' so that the override will take effect for those environments:
- https://github.com/triplea-game/triplea/pull/5852/files#diff-b3216fe9376fa99bf180b01fe4bd1a55R16
- https://github.com/triplea-game/triplea/pull/5852/files#diff-6f7a9a9749d18379ef4b4f7cf267c018R19

Reply:
Thanks for clarifying 👍